src/test/java/edu/harvard/iq/dataverse/export/SchemaDotOrgExporterTest.java

```diff
@@ -408,7 +408,8 @@ private static void mockDatasetFieldSvc() {
             new ControlledVocabularyValue(3l, "bibcode", publicationIdTypes),
             new ControlledVocabularyValue(4l, "doi", publicationIdTypes),
             new ControlledVocabularyValue(5l, "ean13", publicationIdTypes),
-            new ControlledVocabularyValue(6l, "handle", publicationIdTypes)
+            new ControlledVocabularyValue(6l, "handle", publicationIdTypes),
+            new ControlledVocabularyValue(17l, "CSTR", publicationIdTypes)
```

Comment:
Is 171 a typo? I.e. it should be 71? (In this test they just have to be unique though.)